governance/polls/Ratification Poll for Facilitator Onboarding to DIN-001 - @tadeo (MIP41c4-SP35) - June 13, 2022.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Ratification Poll for Facilitator Onboarding to DIN-001 - @tadeo (MIP41c4-SP35) - June 13, 2022
+summary: This subproposal is to onboard @tadeo as Facilitator to the Data Insights Core Unit (DIN-001).
+discussion_link: https://forum.makerdao.com/t/mip41c4-sp35-facilitator-onboarding-din-001/14948
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-06-13T16:00:00
+end_date: 2022-06-27T16:00:00
+---
+# Ratification Poll for Facilitator Onboarding to DIN-001 - tadeo (MIP41c4-SP35) - June 13, 2022
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, June 13 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP41c4-SP35 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*This subproposal adds @tadeo as the Facilitator for the Data Insights Core Unit (DICU).*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/38b7071c8668f2eb20dc35978c59183c924827ac/MIP41/MIP41c4-Subproposals/MIP41c4-SP35.md)
```

Comment:
Use this URL: https://github.com/makerdao/mips/blob/938fcf82d93b708efe850d4e3fa942e3b8440fbc/MIP41/MIP41c4-Subproposals/MIP41c4-SP35.md